addons/mail/static/src/components/thread_preview/thread_preview.xml

```diff
@@ -44,14 +44,34 @@
                         </t>
                     </div>
                     <div class="o_ThreadPreview_core">
-                        <span class="o_ThreadPreview_coreItem o_ThreadPreview_inlineText text-truncate" t-att-class="{ 'o-empty': inlineLastMessageBody.length === 0 }">
-                            <t t-if="threadPreviewView.thread.lastMessage and threadPreviewView.thread.lastMessage.author">
-                                <MessageAuthorPrefix
-                                    messageLocalId="threadPreviewView.thread.lastMessage.localId"
-                                    threadLocalId="threadPreviewView.thread.localId"
-                                />
-                            </t>
-                            <t t-esc="inlineLastMessageBody"/>
+                        <span class="o_ThreadPreview_coreItem o_ThreadPreview_inlineText text-truncate" t-att-class="{ 'o-empty': lastMessageContent.length === 0 }">
+                            <span class="o_ThreadPreview_messageBody">
+                                <t t-if="lastMessageContent.trackingValue">
+                                    <span class="o_ThreadPreview_trackingValue">
+                                        <span class="o_ThreadPreview_trackingValueFieldName">
+                                            <t t-esc="lastMessageContent.trackingValue.changedFieldAsString"/>
+                                        </span>
+                                        <span class="o_ThreadPreview_trackingValueOldValue">
+                                            <t t-esc="lastMessageContent.trackingValue.oldValueAsString"/>
+                                        </span>
+                                        <span class="o_ThreadPreview_arrow">
+                                            <div class="o_ThreadPreview_trackingValueSeparator fa fa-long-arrow-right mx-1" role="img"/>
+                                        </span>
+                                        <span class="o_ThreadPreview_trackingValueNewValue">
+                                            <t t-esc="lastMessageContent.trackingValue.newValueAsString"/>
+                                        </span>
+                                    </span>
+                                </t>
+                                <t t-else="">
+                                    <t t-if="threadPreviewView.thread.lastMessage and threadPreviewView.thread.lastMessage.author">
+                                        <MessageAuthorPrefix
+                                            messageLocalId="threadPreviewView.thread.lastMessage.localId"
+                                            threadLocalId="threadPreviewView.thread.localId"
+                                        />
+                                    </t>
+                                    <t t-esc="lastMessageContent.body"/>
```

Comment:
Maybe this part could also be reused, either in same new component or its own component?